.circleci/config.yml

```diff
@@ -89,9 +90,17 @@ jobs:
           key: v1-gem-cache-{{ checksum "Gemfile.lock" }}
           paths:
             - vendor/bundle
+      - run:
+          name: Install pods
+          command: pod install
+          working_directory: ios/PurchasesHybridCommon
       - run:
           name: Deploy new version
           command: bundle exec fastlane ios release
+      - persist_to_workspace:
+          root: .
+          paths:
+            - PurchasesHybridCommon.framework.zip
```

Comment:
do we need this? it'd already be added to the workspace through the lane, right?

Reply:
This is needed for `make-github-release` which is another job that needs that file. Afaik, just because the lane adds it to the filesystem it doesn't mean it is accessible in a downstream job. Take a look at https://circleci.com/blog/persisting-data-in-workflows-when-to-use-caching-artifacts-and-workspaces/

Reply:
Yeah, this is a problem I had- the file wasn't persisted. Makes sense.